core/src/test/java/org/ehcache/EhcacheBasicGetAllTest.java

```diff
@@ -132,60 +132,26 @@ public void testGetAllNullKey() throws Exception {
    * Tests {@link Ehcache#getAll(Set)} for
    * <ul>
    *    <li>empty request key set</li>
-   *    <li>{@link Store#bulkComputeIfAbsent} throws before accessing loader</li>
+   *    <li>{@link Store#bulkComputeIfAbsent} </li>
    *    <li>no {@code CacheLoaderWriter}</li>
    * </ul>
    */
   @Test
-  public void testGetAllEmptyRequestCacheAccessExceptionBeforeNoLoader() throws Exception {
+  public void testGetAllEmptyRequestNoLoader() throws Exception {
     final FakeStore fakeStore = new FakeStore(Collections.<String, String>emptyMap());
     this.store = spy(fakeStore);
-    doThrow(new CacheAccessException("")).when(this.store)
-        .bulkComputeIfAbsent(getAnyStringSet(), getAnyIterableFunction());
 
     final Ehcache<String, String> ehcache = this.getEhcache(null);
 
     final Map<String, String> actual = ehcache.getAll(Collections.<String>emptySet());
     assertThat(actual, is(notNullValue()));
     assertThat(actual.isEmpty(), is(true));
-
-    verify(this.store).bulkComputeIfAbsent(eq(Collections.<String>emptySet()), getAnyIterableFunction());
-    verify(this.spiedResilienceStrategy)
+    
+    verify(this.store, never()).bulkComputeIfAbsent(eq(Collections.<String>emptySet()), getAnyIterableFunction());
+    verify(this.spiedResilienceStrategy, never())
```

Comment:
Given the verifyZeroInteractions that you inserted aren't these individual verifications redundant?